pkg/features/features.go

```diff
@@ -47,9 +47,13 @@ const (
 
 	UseOSDisk Feature = "UseOSDisk"
 
-	// MountChangeDetection feature flag is used to enable/disable
+	// MountChangeDetection feature flag is used to enable
 	// mount point and fs type change detection for block devices
-	MountChangeDetection = "MountChangeDetection"
+	MountChangeDetection Feature = "MountChangeDetection"
+
+	// SizeChangeDetection feature flag is used to enable size
+	// change detection for block devices
+	SizeChangeDetection Feature = "SizeChangeDetection"
```

Comment:
Instead of using 2 separate feature gates, we should use just one feature gate as both of them are similar